demo/src/main/java/com/eis/geoCalendar/demo/MapsActivity.java

```diff
@@ -19,6 +21,7 @@
 import com.google.android.gms.maps.model.LatLng;
 import com.google.android.gms.maps.model.Marker;
 import com.google.android.gms.maps.model.MarkerOptions;
+import com.eis.geoCalendar.demo.Localization.*;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Using the '.*' form of import should be avoided - com.eis.geoCalendar.demo.Localization.*.](https://app.codacy.com/manual/CremaLuca/geoCalendar/pullRequest?prid=4824932)